package.json

```diff
@@ -6,13 +6,13 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "mocha -- out/test/*.js",
-    "pretest": "eslint ./src/*.ts && npm run compile",
-    "build": "tsc",
-    "compile": "tsc && npm run copyToDist",
-    "copyToDist": "copyfiles --error --verbose --flat ./out/src/*.* ./dist/",
-    "prepack": "npm run test && npm run copyToDist",
-    "watch": "tsc -w"
+    "test": "npx mocha -- out/test/*.js",
+    "pretest": "npx eslint src --ext ts && npm run compile",
+    "build": "npx tsc",
```

Comment:
Normally I was depending on it through the NPM registry directly, which worked fine, but the failing case was when I added a direct dependency on my github fork:

```json
    "dependencies": {
        "vscode-extension-updater": "github:jorgenpt/vscode-extension-updater"
    }```

That did not work (presumably because I didn't have the correct packages installed globally). I was able to reproduce it by doing `npm --prefix c:\dev\vscode-extension-updater test`, and that's also how I verified my changes.

Reply:
Here's my local test on `mocha` in my local checkout of the fork.

Without npx:

```
$ npm run simple

> vscode-extension-updater@1.1.1 simple C:\dev\vscode-extension-updater
> mocha --version

'mocha' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vscode-extension-updater@1.1.1 simple: `mocha --version`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vscode-extension-updater@1.1.1 simple script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jorgenpt\AppData\Roaming\npm-cache\_logs\2021-08-02T21_27_20_985Z-debug.log
```

With npx:

```
$ npm run simple_npx

> vscode-extension-updater@1.1.1 simple_npx C:\dev\vscode-extension-updater
> npx mocha --version

npx: installed 92 in 3.991s
9.0.3
```

In this example I added two scripts to the `package.json`:
```json
  "scripts": {
    "simple": "mocha --version",
    "simple_npx": "npx mocha --version"
  },
```